sqlmesh/cli/main.py

```diff
@@ -287,14 +287,27 @@ def diff(ctx: click.Context, environment: t.Optional[str] = None) -> None:
     multiple=True,
     help="Select specific model changes that should be included in the plan.",
 )
+@click.option(
+    "--backfill-model",
```

Comment:
plurals?

Reply:
No, see `--select-model` and `--restate-model`. Multiple entires in CLI look like following:
```
sqlmesh plan --backfill-model model_a --backfill-model "<expression>"
```